### PR TITLE
Switch to iso2 codes

### PR DESCRIPTION
**Pull request by helocordi**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch
- [x] I have tested my code with the `xcodebuild` and `swiftlint --strict` commands as directed in the [testing section of the contributing guide](https://github.com/scribe-org/Scribe-iOS/blob/main/CONTRIBUTING.md#testing)

---

### Description

In response to issue #494, I've switched every single reference to a language to ISO-2 code. 
  "Danish": "da",
  "English": "en",
  "French": "fr",
  "German": "de",
  "Italian": "it",
  "Norwegian": "nb",
  "Portuguese": "pt",
  "Russian": "ru",
  "Spanish": "es",
  "Swedish": "sv"
  
It still needs to be tested a bit to ensure that the change doesn't lead to new errors.

### Related issue

- #494

### CODE REVIEW

File: Keyboards/KeyboardsBase/InterfaceVariables.swift [hunk 1]
```diff
@@ -162,25 +162,27 @@ var controllerLanguage = String()
...
 let languagesStringDict = [
-  "English": NSLocalizedString("app._global.english", value: "English", comment: ""),
-  "French": NSLocalizedString("app._global.french", value: "French", comment: ""),
-  "German": NSLocalizedString("app._global.german", value: "German", comment: ""),
-  "Italian": NSLocalizedString("app._global.italian", value: "Italian", comment: ""),
-  "Portuguese": NSLocalizedString("app._global.portuguese", value: "Portuguese", comment: ""),
-  "Russian": NSLocalizedString("app._global.russian", value: "Russian", comment: ""),
-  "Spanish": NSLocalizedString("app._global.spanish", value: "Spanish", comment: ""),
-  "Swedish": NSLocalizedString("app._global.swedish", value: "Swedish", comment: "")
+  "EN": NSLocalizedString("app._global.en", value: "EN", comment: ""),
```
[the remark this answers is not in the document]
Comment (helocordi) [reply Keyboards/KeyboardsBase/InterfaceVariables.swift:178]: > These lines here need to be reverted to what they were as they're referencing data coming from Scribe-i18n :) No stress as there's little way for you to know this, @helocordi!

Ok I'll change that

**Comment by andrewtavis** [comment]
Initial glances are looking really good, @helocordi! We'll try to bring this in in the comings days :) Can you fix the comment above?

**Comment by helocordi** [comment]
> Initial glances are looking really good, @helocordi! We'll try to bring this in in the comings days :) Can you fix the comment above?

Perfect, I've changed it now :)

**Review by henrikth93 — COMMENTED** [review]
Looks good to me

**Review by andrewtavis — APPROVED** [review]
Ultimately we do did need to revert the changes to the controller language references as I forgot that those are coming directly from the settings for each keyboard. We could switch it over, but then in thinking as well there might be some pros to the full language as strings as there will one day be languages that people are not sure on for the iso-2 code 😊 As it stands now, we're changing variable names to just use iSO-2 codes, which further matches the file names. I'd say this is great 😊 

Thanks for the great contribution, @helocordi!

**Pull request merged**